assets/src/edit-story/app/media/mediaProvider.js

```diff
@@ -56,8 +56,17 @@ function MediaProvider({ children }) {
   const {
     actions: { getMedia },
   } = useAPI();
+
+  // Used by callbacks to avoid unnecessary dependency on state changes,
+  // which cause re-renderings.
+  const stateRef = useRef();
```

Comment:
Can't you pass state to the function directly?

Reply:
Yeah, `initialValue` can be passed

Reply:
If only the initialValue is assigned, then how would the state be updated when it changes?